### ISSUE TITLE
[Feature] Add more variables to command or url action

### ISSUE THREAD

**Issue by e1ze** [HenriWahl/Nagstamon#375]
Hi,

we have a lot of custom actions defined for accessing our ticket system, pnp4nagios, splunk, icinga history, dokuwiki (documentation for host/services) etc. Theses actions are filled with your placeholder variables. I am missing some variables for usage in our email template for our front desk. They send a lot of custom emails from our monitoring system, so i have created a standard email template with automatically generated and fixed information.
Is it possible to create some new variables for icinga/nagios:

- duration
- last check
- status
- attempt

Basically for all information that you can see in the nagstamon gui or if you **right click** an event and use **Copy to Clipboard - All Information**:
Example: 

Host: xxxx
Service: xxxx
**Status: CRITICAL
Last check: 04-07-2017 07:58:24
Duration: 0d  0h  7m 57s
Attempt: 3/3**
Status information: CRITICAL - abcde - 123

Thanks

Greetings
René

**Comment by HenriWahl** [comment]
Good idea.

**Comment by e1ze** [comment]
Hi Henri,

I wanted to ask you if you have made any progress with this feature request so far?

Thanks,
René

**Comment by HenriWahl** [comment]
Due to a bunch of other non-Nagstamon-stuff I am sorry to say no, there is no progress. :-(
